### ISSUE TITLE
No suspicions found on election_expenses classifier

### ISSUE THREAD

**Issue by cuducos** [datasciencebr/rosie#28]
Not sure if it's a bug or if I'm doing something wrong. But @Irio shared a brand new `irregularities.xz` in PVT this week:

```
>>> import pandas as pd
>>> df = pd.read_csv('…/irregularities.xz')
>>> df[df.election_expenses == True]
Empty DataFrame
Columns: [applicant_id, year, document_id, meal_price_outlier, invalid_cnpj_cpf, suspicious_traveled_speed_day, over_monthly_subquota_limit, election_expenses]
Index: []
```

Is that expected?

cc @jtemporal

**Comment by fariaslcfs** [comment]
Would it be this way?
 df[ df[ 'election_expenses' ] == True]

**Comment by cuducos** [comment]
It's exactly the same thing, @fariaslcfs: https://pandas-docs.github.io/pandas-docs-travis/indexing.html#attribute-access

**Comment by fariaslcfs** [comment]
@cuducos Sorry for asking this, but have you checked (manually) the contents of the irregularities file? Are there any 'True' records in it? I checked one local irregularities file and all the syntaxes worked fine.
df[df['column_name'] == value]
df[df.column_name == value]
df.loc[df.column_name == value]
df.loc[df['column_name'] == value]

**Comment by cuducos** [comment]
> Are there any 'True' records in it?

Yes:

```
>>> import pandas as pd
>>> df = pd.read_csv('irregularities.xz')
>>> len(df[df.meal_price_outlier==True])
2127
>>> len(df[df.invalid_cnpj_cpf==True])
16
>>> len(df[df.suspicious_traveled_speed_day==True])
786
>>> len(df[df.over_monthly_subquota_limit==True])
15
>>> len(df[df.election_expenses==True])
 0
```

**Comment by fariaslcfs** [comment]
No election_expenses classified as True, so it's ok. Nothing wrong about it.

**Comment by cuducos** [comment]
> No election_expenses classified as True, so it's ok. Nothing wrong about it.

Why? According to [this notebook](https://github.com/datasciencebr/serenata-de-amor/blob/master/develop/2016-12-07-irio-election-expenses.ipynb) I was expecting at least 47 suspicions from the `election_expenses` classifier.

**Comment by jtemporal** [comment]
@cuducos This is not expected. Same thing happened to the #27 =/

What I'm going to do right now is: re-run the notebooks with the new datasets to check if it's outcome changes or not. As soon as I get any findings I'll post it here.

**Comment by jtemporal** [comment]
Ok so, possibly all 47 suspicions from `election_expenses` classifier were reported. With a new reimbursements data set generated today, and the notebook updated I got 12 suspicions. So for sure not zero.
From invalid companies I also re-run the notebook and it still gave me 5222. Further investigations need to be done before we can close this one

**Comment by cuducos** [comment]
> So for sure not zero.

🎉

**Comment by jtemporal** [comment]
So some news for you guys...
<img width="472" alt="screen shot 2017-04-03 at 10 05 42 pm" src="https://cloud.githubusercontent.com/assets/6595551/24638375/39355734-18be-11e7-9a41-ff9189f5061e.png">
The problem was indeed the dataset.
So, I already shared the new irregularities file with @cuducos and I'm now closing this issue ;)

**Comment by cuducos** [comment]
What actually was the problem? Was it the problem described in #30? Or in other words: what did actually close this issue?

**Comment by cuducos** [comment]
Ow… ok, now I see. Closed in favor of #30 indeed!

**Comment by jtemporal** [comment]
So just to be clear:
>What actually was the problem? 

The company was being queried out by the missing coordinates 

now that's a proper explanation ;P
